addons/test_mail/tests/test_mail_composer.py

```diff
@@ -1056,7 +1056,7 @@ def test_mail_composer_recipients(self):
                 self.assertEqual(composer.partner_ids, base_recipients)
                 self.assertEqual(composer.reply_to, 'my_reply_to@test.example.com')
                 self.assertFalse(composer.reply_to_force_new)
-                self.assertEqual(composer.reply_to_mode, 'update')
+                self.assertEqual(composer.reply_to_mode, 'new')
```

Comment:
Mmh I thought that with the inverse method,`reply_to_force_new` would also be set to true.
Isn't that an issue that `reply_to_force_new` can be False while `reply_mode` is True?
Maybe we should instead manually set `reply_to_force_new` when applying a template that has a `reply_to`?
What do you think?